Rakefile

```diff
@@ -3,6 +3,10 @@ require "rake/testtask"
 require "rbconfig"
 require 'rake/extensiontask'
 
+on_windows = /mswin|mingw/ =~ RUBY_PLATFORM
```

Comment:
It seems there are more patterns, could we use `Gem.win_platform?` instead? https://github.com/rubygems/rubygems/blob/master/lib/rubygems.rb#L1028-L1035